includes/pages/admin_questions.php

```diff
@@ -153,13 +147,11 @@ function admin_questions()
                     return error('Incomplete call, missing Question ID.', true);
                 }
 
-                $question = DB::selectOne(
-                    'SELECT * FROM `Questions` WHERE `QID`=? LIMIT 1',
-                    [$question_id]
-                );
+                $questions =   Question::where(['id' => $question_id]);
```

Comment:
One whitespace after the `=` is enough ;)

Reply:
haha, one can never have too many whitespaces :P

Reply:
Ährm ufff